<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="classes" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="com.naveenautomation.Listeners.CustomListener"/>
    <listener class-name="com.naveenautomation.Reports.ExtendReportListener"/>
    <listener class-name="com.naveenautomation.Utils.TransformerForRetry"/>
  </listeners>
  <test thread-count="5" parallel="classes" name="Test2(failed)">
    <classes>
      <class name="com.naveenautomation.tests.MyAccountInformationTest">
        <methods>
          <include name="setUp"/>
          <include name="VerifyUsersAreAbleToLoginSuccessFully" invocation-numbers="0"/>
          <include name="teardown"/>
        </methods>
      </class> <!-- com.naveenautomation.tests.MyAccountInformationTest -->
    </classes>
  </test> <!-- Test2(failed) -->
</suite> <!-- Failed suite [Suite] -->
